lib/Composers/Composers.dart

```diff
@@ -27,7 +27,7 @@ class ComposersScreen extends StatelessWidget {
                   padding: const EdgeInsets.all(40),
                   child: Column(
                     children: <Widget>[
-                      const Text('Composers', style: TextStyle(fontSize: 40)),
+                      const Text('Composers', style: TextStyle(fontSize: 80)),
```

Comment:
80 is a little too big for the text on some screens, 60 should be a happier medium I think

![image](https://user-images.githubusercontent.com/114765630/233649649-f6d0e442-5e8a-4ba1-9231-25c930baab11.png)